### DISCUSSION TITLE
Have option in smallcalc_applet@lerc.sds for limiting the decimals

### DISCUSSION THREAD

**Discussion by Kellsen** [linuxmint/discussions#193] [Ideas]
What I first thought was a bug turned out to be a feature: that the results in smallcalc are always calculated to decimals even when they (imo) shouldn't.
I.e.:
![274739858-5c42a287-0533-4f43-b980-dc2b50532087](https://github.com/linuxmint/discussions/assets/93977235/dc75bbd1-7133-4559-a335-dde75015bb82)

gives

![274739864-a8a88a92-8e23-45d5-b038-af998c4ddead](https://github.com/linuxmint/discussions/assets/93977235/cd4616c8-68ab-4410-ac2e-670088cbc4d9)

Or

![274739459-d5d1d7a7-26ad-48eb-9db7-0cf5f82d7d59](https://github.com/linuxmint/discussions/assets/93977235/9a405d1e-7a0c-4c8e-a56c-5f759d5e35b7)

results in

![274739484-cc025e45-bac2-4230-9bdc-9912b3c06485](https://github.com/linuxmint/discussions/assets/93977235/abae975c-34e3-44c0-b207-7baf0e5ae917)

This clutters up the result and is confusing when you expect a number with one, maybe two decimal places and when it obstructs the first digit so you won't see the entire result.
I for one won't be doing any complicated math, more easy additions or multiplications. So for me it'd be sufficient to limit the result to be rounded to two or three or even no decimal places.
If there is already a way to adjust that in the math.js script, apologies but please let me know.

**Comment by claudiux** [comment]
@Kellsen 
Please take a look to https://github.com/linuxmint/cinnamon-spices-applets/pull/5049

**Comment by claudiux** [comment]
@clefebvre 
This discussion can be closed.
